youtube_dl/utils.py

```diff
@@ -91,6 +91,10 @@ def register_socks_protocols():
     'January', 'February', 'March', 'April', 'May', 'June',
     'July', 'August', 'September', 'October', 'November', 'December']
 
+FRENCH_MONTH_NAMES = [
```

Comment:
@TRox1972 regarding FRENCH_MONTH_NAMES, I have made like ENGLISH_MONTH_NAMES.
Tell me what is the best way and I'll commit the correction if necessary.
Thanks a lot